src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random randomColor = new Random();
```

Comment:
It is not a color yeat, is it?)
```suggestion
        Random random = new Random();
```